Taskfile.yml

```diff
@@ -63,7 +63,6 @@ tasks:
       - go mod tidy
       - go mod verify
       - task: gen-version
-      - task: build-envtool
```

Comment:
That part was merged in another PR, and now there is an unresolved merge conflict